scylla/src/transport/connection.rs

```diff
@@ -1464,23 +1464,30 @@ mod tests {
         let (connection, _) = super::open_connection(addr, None, ConnectionConfig::default())
             .await
             .unwrap();
+        let ks = crate::transport::session_test::unique_name();
 
-        connection.query_single_page("CREATE KEYSPACE IF NOT EXISTS ks WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor' : 1}", &[]).await.unwrap();
+        connection.query_single_page(format!("CREATE KEYSPACE IF NOT EXISTS {} WITH REPLICATION = {{'class' : 'SimpleStrategy', 'replication_factor' : 1}}", ks), &[]).await.unwrap();
         connection
-            .query_single_page("DROP TABLE IF EXISTS ks.connection_query_all_tab", &[])
+            .use_keyspace(&super::VerifiedKeyspaceName {
```

Comment:
There is a `VerifiedKeyspaceName::new` constructor which verifies the correctness of the keyspace name. `VerifiedKeyspaceName` shouldn't be constructed directly because it bypasses the check. It shouldn't be possible for the users of the crates because of the `name` field visibility, I'm actually surprised it's possible to do it here.

Reply:
ok, let me fix the usage

Reply:
done